lib/fcrepo_wrapper/configuration.rb

```diff
@@ -77,15 +81,10 @@ def java_options
     end
 
     def default_java_options
-      ['-Dfcrepo.log.http.api=WARN',
-      # To avoid "WARN: The namespace of predicate:
-      # info:fedora/fedora-system:def/relations-external#isPartOf
-      # was possibly misinterpreted as:
-      # info:fedora/fedora-system:def/relations-external#."
-      '-Dfcrepo.log.kernel=ERROR',
-      ("-Dfcrepo.home=#{fcrepo_home_dir}" if fcrepo_home_dir),
-      ("-Dfcrepo.spring.jms.configuration=#{spring_noop_file}" unless jms_enabled?),
-      '-Xmx512m'].compact
+      ["-Dfcrepo.log=#{log_level}",
```

Comment:
Forcing all the logging properties to be the same isn't ideal, but it seems like the best trade-off of simplicity and configurability.